### PR TITLE
chore(release): 5.2.7

### PR DESCRIPTION
**Pull request by pbabz**
Issue #, if available:

Description of changes:
Update changelog for 5.2.7 release

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,16 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+<<<<<<< HEAD
```
Comment (kpark277) [inline CHANGELOG.md]: Delete merge conflict lines.

**Review by kpark277 — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -281,16 +284,10 @@ To learn more about the new process, refer to the updated [instructions](https:/
 
 **Important Notes:**
 
-<<<<<<< HEAD
-
 - If you have already onboarded a member account for your Service Workbench installation, and this account has active or stopped workspaces, the safest course would be to terminate all workspaces prior to the update. We did test a scenario with active and stopped workspaces and observed no impact during testing, but because this update is a major release, we recommend the safest course.
-- # Any member accounts that were onboarded prior to this update will need to be updated through the Service Workbench user interface, and you will be prompted to do so when visiting the new “Accounts” page in Service Workbench. This update is necessary because there is a new capability that will check to see if the member and main account code versions are in sync, and provide a visual indicator if not, allowing you a clear indication of update.
```
Comment (nguyen102) [inline CHANGELOG.md:287]: Can we confirm this deletion is correct?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,8 +2,11 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## Beta
-[This release is in beta. Click here to see changes since 5.2.6.](https://github.com/awslabs/service-workbench-on-aws/compare/v5.2.6...mainline)
+### [5.2.7](https://github.com/awslabs/service-workbench-on-aws/compare/v5.2.6...v5.2.7) (2022-12-30)
+
+### Bug Fixes
+
+* Sagemaker compatability fix with AL2. Fix issue in which Sagemaker instances running on AL2 could not mount dataset. ([#1089](https://github.com/awslabs/service-workbench-on-aws/issues/1089)) ([bdf978b](https://github.com/awslabs/service-workbench-on-aws/commit/bdf978b1c909c37207902f1b29b0fd05b5b24208))
```
Comment (nguyen102) [inline CHANGELOG.md]: Nitpick: compatability is spelled wrong.

File: CHANGELOG.md (hunk 2 above)
Comment (pbabz) [reply CHANGELOG.md:287]: It's a duplicate from a poor merge conflict resolution before. I confirmed the text was the same. Main differences was bullet style, and a random # sign

**Review by kpark277 — APPROVED** [review]

**Pull request merged**